apps/camera/js/views/controls.js

```diff
@@ -239,11 +252,11 @@ module.exports = View.extend({
   template: function() {
     /*jshint maxlen:false*/
     return '<div class="controls-left">' +
-      '<div class="controls-button controls-thumbnail-button test-thumbnail js-thumbnail rotates" name="thumbnail"></div>' +
+      '<div class="controls-button controls-thumbnail-button test-thumbnail js-thumbnail rotates" name="thumbnail" role="button" data-l10n-id="thumbnail-button" aria-label="Photo Thumbnail"></div>' +
       '<div class="controls-button controls-cancel-pick-button test-cancel-pick rotates js-cancel" name="cancel" data-icon="close"></div>' +
     '</div>' +
     '<div class="controls-middle">' +
-      '<div class="capture-button test-capture rotates js-capture" name="capture">' +
+      '<div class="capture-button test-capture rotates js-capture" name="capture" role="button" data-l10n-id="capture-button" aria-label="Shutter">' +
```

Comment:
Should the `aria-label` be on here? I thought l10n.js took care of adding these attributes?